pkg/detectors/mongodb/mongodb.go

```diff
@@ -85,6 +87,30 @@ func isErrDeterminate(err error) bool {
 }
 
 func verifyUri(uri string, timeout time.Duration) error {
+	parsed, err := url.Parse(uri)
+	if err != nil {
+		return err
+	}
+
+	params := url.Values{}
+	for k, v := range parsed.Query() {
+		if len(v) > 0 {
+			switch k {
+			case "tls":
+				if v[0] == "false" {
+					params.Set("tls", "false")
+				} else {
+					params.Set("tls", "true")
+				}
+			}
+		}
+	}
```

Comment:
**Suggestion:**

```go
if v := parsed.Query().Get("tls"); v == "false" {
    params.Set("tls", "false")
} else if v != "" {
    params.Set("tls", "true")
}
```

Reply:
I added this in a way to easily add more parameters in the future, even if it's not the most efficient.